trackpy/framewise_data.py

```diff
@@ -248,13 +248,13 @@ def __init__(self, filename, key='FrameData', mode='a', t_column='frame',
         self._t_column = t_column
         self.store = pd.HDFStore(self.filename, mode, **kwargs)
 
-        with pd.get_store(self.filename) as store:
-            try:
-                store[self.key]
-            except KeyError:
-                pass
-            else:
-                self._validate_node(use_tabular_copy)
+        store = pd.HDFStore(self.filename)
+        try:
+            store[self.key]
+        except KeyError:
+            pass
+        else:
+            self._validate_node(use_tabular_copy)
```

Comment:
@rbnvrw It appears that adding `store.close()` below this line may fix the Windows error. You can try it yourself, or I will try to push that change this weekend.

Reply:
Yes, this makes sense because it was a `with` statement before but that syntax is not supported anymore in the new pandas implementation of HDFStore. I have added the line.